options/ansi/include/string.h

```diff
@@ -51,8 +51,11 @@ void *memset(void *dest, int c, size_t size);
 char *strerror(int errnum);
 size_t strlen(const char *s);
 
-// POSIX extensions.
+#if defined(_BSD_SOURCE) || defined(_GNU_SOURCE)
```

Comment:
should this not instead check if the POSIX option is enabled?

Reply:
Yeah this should also check if the the header is enabled, but the feature test macros are correct here.